templates/rbac.yaml

```diff
@@ -68,17 +26,11 @@ rules:
       - "" # indicates the core API group
       - "apps"
     resources:
-      # Every environment deployment has a service.
-      - services
       - pods
       # Required for creating registry pull secrets.
       - secrets
-      # Required for the code-server volume and the environments' home volumes.
+      # Required for the environments' home volumes.
       - persistentvolumeclaims
-      # Required for legacy reasons.
-      - statefulsets
-      # Every environment is a deployment.
-      - deployments
```

Comment:
We still have deployments code for migrating legacy deployments to pods, so this can't be removed. Pods are fairly new so I'm not comfortable removing that code yet